### PR TITLE
Header mobile

### LINKED ISSUE #1596: Standardize Program Areas page header in mobile
### Overview
We need to standardize the Program Areas page header in mobile to make it consistent with other pages and our design system.

### Action Items
- [x] Standardize the Program Areas page header in mobile (see annotated screenshot below)
   - [x] Decrease the margin between heading and top nav bar to 32px (2rem)- this should be done by just removing the margin-top: 2.25rem) - **only in mobile**
   - [x] Increase margin between heading and paragraph to 28px total - **only in mobile**
   - [x] Increase margin between header paragraph and illustration to 36px total - **only in mobile**
   - [x] Decrease the size of the header illustration to max-width 256px - **only in mobile**

### Resources/Instructions
[Program Areas page](https://www.hackforla.org/program-areas)

**How header looks now (incorrect)**

Screen Shot 2021-05-20 at 12 54 29 AM

**How header should look (correct):**

Screen Shot 2021-05-20 at 12 54 14 AM

### PR DESCRIPTION
**Pull request by Abel-Zambrano**
Fixes #1596 

  - Decreased the margin between heading and top nav bar to 32px (2rem) - only in mobile
  - Increased margin between heading and paragraph to 28px total - only in mobile
  - Increased margin between header paragraph and illustration to 36px total - only in mobile
  - Decreased the size of the header illustration to max-width 256px - only in mobile

### Screenshots of Proposed Changes Of The Website
<details>
<summary>Changes</summary>
<summary>Before</summary>
Screen Shot 2021-06-29 at 5 57 44 PM
<summary>After</summary>
Screen Shot 2021-06-29 at 5 59 30 PM

Paste_Your_Image_Link_Here

</details>

### CODE REVIEW

**Review by Zak234 — COMMENTED** [review]
There seems to be some problems with the issue as #1596 and #1825 seem to both be requesting similar changes to the header illustration. Additionally, the margin in between the paragraph text and header illustration seems to be much larger than shown in the mock up photo shown in #1596 and I think this may be due to you using rem instead of px but I am unsure.

**Comment by Aveline-art** [comment]
Per @Zak234 's comment, #1596 and #1825 appears to be overlapping issues. @daniellex0 Can you clarify if one issue supersedes the other or if both should be reviewed and merged?

File: _sass/components/_program-areas.scss [hunk 1]
```diff
@@ -55,6 +63,10 @@
 .program-areas-hero-svg {
     width: auto;
     height: 240px;
+
+  @media #{$bp-below-tablet} {
+      max-width: 256px;
```
Comment (jbubar) [inline _sass/components/_program-areas.scss]: could you actually use percentage here? use `max-width: 70%`

**Review by jbubar — CHANGES_REQUESTED** [review]
@Abel-Zambrano you did a great job with this one. you did everything written on the issue. I do have one request. 

Could you use percentage for the max-width of the image, instead of pixes?
I know the issue says otherwise, but if you do 70% it should be around 256px; for the small view that danielle had in the issue. 
<details>
    <summary>how it looks on resizing as compared to the join us page</summary>

abel's-css-thing


</details>

**Comment by jbubar** [comment]
@Aveline-art @Zak234. Thanks for noticing the redundancy. I took the liberty to separate out the issues and issue #1825 should not include this. @daniellex0 I hope this is ok.

**Comment by Abel-Zambrano** [comment]
@jbubar the requested changes are complete.

**Comment by daniellex0** [comment]
@jbubar yup that's fine!

File: _sass/components/_program-areas.scss [hunk 2]
```diff
@@ -55,6 +63,10 @@
 .program-areas-hero-svg {
     width: auto;
     height: 240px;
+
+  @media #{$bp-below-tablet} {
+      max-width: 70%;
```
Comment (ghost) [inline _sass/components/_program-areas.scss:68]: Made the changes requested @jbubar

**Comment by jbubar** [comment]
@Abel-Zambrano this is ready to merge, but there are some minor merge conflicts, Let me know if you need a hand with it

**Comment by Abel-Zambrano** [comment]
@jbubar I resolved the conflicts, ready for review.

**Comment by jbubar** [comment]
@Abel-Zambrano This looks ready to merge, but there was a mistake with another merge, so I want to revert that and check before merging this.. You can feel free to pick up another issue

**Comment by jbubar** [comment]
sorry @Abel-Zambrano, there are merge conflict again. there was a mix up with another issue, which should not have been merged and needed to be reverted.

**Comment by jbubar** [comment]
It caused the merge conflicts the first time, and it is causing them again because we reverted it

**Comment by Abel-Zambrano** [comment]
@jbubar conflicts fixed.

**Review by jbubar — CHANGES_REQUESTED** [review]
Hey. did you notice the duplicate code? Screen Shot 2021-07-08 at 7 29 01 PM
 also there is a missing "}"

**Comment by Abel-Zambrano** [comment]
I removed the duplicate code. @jbubar

File: _sass/components/_program-areas.scss [hunk 3]
```diff
@@ -55,7 +63,10 @@
 .program-areas-hero-svg {
```
Comment (jbubar) [inline _sass/components/_program-areas.scss:63]: The opening bracket on this line is never closed.

**Review by jbubar — CHANGES_REQUESTED** [review]

File: _sass/components/_program-areas.scss (hunk 3 above)
Comment (jbubar) [reply _sass/components/_program-areas.scss:63]: <details>
    <summary>error that I got when I ran `docker compose up`</summary>

Screen Shot 2021-07-10 at 11 38 19 AM


</details>

**Pull request closed without merging**